static/js/components/subject_page/block.tsx

```diff
@@ -235,17 +318,21 @@ function renderTiles(
       : props.place;
     const comparisonPlaces = getComparisonPlaces(tile, place);
     const className = classNameList.join(" ");
+    const statVarSpecs = props.statVarProvider.getSpecList(tile.statVarKey, {
```

Comment:
curious why you moved this here even though most of the switch statements are still making this call?